app/controllers/comment_controller.rb

```diff
@@ -8,8 +8,12 @@ def create
         comment = Comment.new(text: values[:text])
         comment.author_id = user.id
         comment.post_id = post.id
-
-        redirect_to "/user/#{user.id}/post/#{post.id}" if comment.save
+        if comment.save
+          flash[:success] = 'Comment added correctly'
+        else
+          flash[:error] = 'Error! comment not added'
+        end
+        redirect_to "/user/#{user.id}/post/#{post.id}"
```

Comment:
- [ ] Well done :clap: on adding the flash message on the comment creation however you still need to add also aflash message when the like is created and of course render them in the view as required in the project guidelines.

Reply:
Hi @andersonlebon, thank you for your review. 
I think that adding a message when the likes are added is kind of weird since how do you test if it wasn't added?
it just increments the counter so that should be a proof of what is happening 

I would add by the way I just that I see that feature as redundant. thanks a lot 😄